file.go

```diff
@@ -1,37 +1,204 @@
 package jcsv
 
-import "os"
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	data []map[string]interface{}
 }
 
 func ParseJsonFile(path string) (file, error) {
-	// TODO: open and read the given file into your `file` object
-	return file{}, nil
+	//open JSON file
+	jsonFile, err := os.Open(path)
+
+	if err != nil {
+		fmt.Println(err)
+		return file{}, err
+	}
+
+	defer jsonFile.Close()
+
+	//read complete json file and store in byteValue
```

Comment:
```suggestion
	// read complete json file and store in byteValue
```
Do this is all comments.

Reply:
all right